kubectl/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM debian:stable-slim AS release
```

Comment:
Why not an alpine docker image?

Reply:
I would've had to build Kubectl or hack Alpine with a Glibc, since there is no binary for Alpine. Thus I decided for the sake of brevity to use Debian.